### ISSUE TITLE
Creating a MySQL database to store tables for all diagnosis and symptom information for each user, relying on input from Django website models

### ISSUE THREAD

**Issue by oscuml** [oscuml/Diagknow#4]
This is another task that involves making multiple tables within a database, but each table will be for every user that has registered and entered their symptoms into the diagnosis page for our website.  

1.   :white_check_mark:  Each user will have a table for their symptoms, their possible diseases arising from each symptom, and how long each symptom has appeared.  This is essentially taking all the information each user enters into the diagnosis page and storing all users info together.
2.   :white_check_mark:  There must also be a column saying whether or not they have been diagnosed or not.  This will be manually entered by a website operator at first, but should be manual afterward.  
3.   Completing this task is contingent on whether or not the Django model instance issue has been completed.
4.   :white_check_mark: When the user enters their information into the web page, the Django instance should be created on our Python Django interface, and the website operator can manually run our main disease determination algorithm on the user.  
      :white_check_mark: When the results have come back, the user can first manually enter them and the status of diagnosis into this database, but after we integrate the Django input to PHP, this will be manually done by the website.
5.  To test the database, we can use a placeholder function to determine the disease for each symptom so that the data can be entered into the database.
6.  Example table:
 |User: Last, First | <br>
|       Symptoms       |            Symptom occurrence          |       Time of occurrence     |         Disease          |
_____________________________________________________________________________________________________________
|       Symptom 1     |             How long                            |                    When             |   Possible diseases |
_____________________________________________________________________________________________________________
|       Symptom 2     |             How long                            |                    When             |   Possible diseases |
_____________________________________________________________________________________________________________
|       Symptom 3     |             How long                            |                    When             |   Possible diseases |
_____________________________________________________________________________________________________________
|       Symptom 4     |             How long                            |                    When             |   Possible diseases |
_____________________________________________________________________________________________________________
|       Diagnosed?     |                                                        |      Time of diagnosis        |        Result              |
